qa/tasks/cephfs/test_failover.py

```diff
@@ -298,6 +298,25 @@ class TestFailover(CephFSTestCase):
     CLIENTS_REQUIRED = 1
     MDSS_REQUIRED = 2
 
+    def test_repeated_boot(self):
+        """
+        That multiple boot messages do not result in the MDS getting evicted.
+        """
+
+        interval = 10
+        self.config_set("mon", "paxos_propose_interval", interval)
+
+        mds = choice(list(self.fs.status().get_all()))
+
+        with self.assert_cluster_log(f"daemon mds.{mds['name']} restarted", present=False):
```

Comment:
oh, you meant `present=True`?

https://pulpito.ceph.com/vshankar-2023-04-21_04:40:06-fs-wip-vshankar-testing-20230420.132447-testing-default-smithi/7247251/

Reply:
No. The test had a different behavior in teuthology. I've adjusted it so it should now work on vstart_runner and teuthology.